lab_1/main.py

```diff
@@ -70,60 +79,22 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
-
-
-def get_adjacent_words(tokens: list, word: str, left_n: int, right_n: int) -> list:
```

Comment:
do not delete functions even if you are not going to implement them
change back